src/viam/examples/project/pkg-config/Makefile

```diff
@@ -1,19 +1,24 @@
 CXX = c++
 
-CXXFLAGS = -std=c++14
-CXXFLAGS += `pkg-config --cflags viam-cpp-sdk-libviamsdk`
-LINKFLAGS = `pkg-config --libs viam-cpp-sdk-libviamsdk`
+VIAM_PREFIX ?= ../../../../../build/install
+PKG_CONFIG_PATH ?= $(VIAM_PREFIX)/lib/pkgconfig
+
+
+CXXFLAGS = -std=c++17
+CXXFLAGS += `PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --cflags viam-cpp-sdk-libviamsdk`
+LINKFLAGS = `PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --libs viam-cpp-sdk-libviamsdk`
+LDFLAGS ="-Wl,-rpath,$(VIAM_PREFIX)/"
```

Comment:
Why introduce a new `LDFLAGS` rather than just appending to `LINKFLAGS`?

Also, consider just using `LDFLAGS` for this, since that is the autoconf name for link flags